units/debug-shell.service.in

```diff
@@ -17,7 +17,7 @@ After=systemd-vconsole-setup.service
 
 [Service]
 Environment=TERM=linux
-ExecStart={{SUSHELL}}
+ExecStart=-{{SUSHELL}}
```

Comment:
This should be a separate commit with explanation?

Reply:
I guess for the old use case this does not matter, but now the exit code of the last command run before existing the shell is propagated to the `debug-shell.service`, so if it's non-zero, the service appears with a failed state.

Reply:
Makes sense, but please add a comment to the unit file with this explanation